src/components/new-request/name-check/name-check-conflicts.vue

```diff
@@ -0,0 +1,320 @@
+<template>
+  <v-data-table :headers="headers"
+                fixed
+                hide-default-header hide-default-footer
+                :items="items"
+                item-key="problem"
+                single-expand
+                :expanded.sync="expanded"
+                no-data-text=""
+                show-expand>
```

Comment:
I don't mind this layout, but i think* we've been going with the next line with 1 tab for props. Up to you though!

Reply:
ha, okay I've been wondering what the best way for this is. I've done it this way everywhere... Is it okay if I leave it so it's consistent throughout the namecheck vues? Otherwise I will have a lot of code to go though haha

Reply:
Totally, of course :) I'm easy!